bukuserver/templates/bukuserver/home.html

```diff
@@ -4,14 +4,16 @@
 {% block head %}
   {{ super() }}
   {{ buku.close_if_popup() }}
-  {{ buku.focus('form[action="/"]') }}
+  {{ buku.focus('main form[action="/"]') }}
```

Comment:
Both forms have the same `action` now, thus specifying which one we want focused on page load.